.github/workflows/ci.yaml

```diff
@@ -108,15 +110,33 @@ jobs:
       fail-fast: false
       matrix:
         k8s: [v1.23.13, v1.27.1]
-        ARCH: [amd64]
+        ARCH: [amd64, arm64]
         include:
           - ARCH: amd64
             RUNNER: self-hosted-1cpu
+          - ARCH: arm64
+            RUNNER: self-hosted-arm64-1cpu
     env:
       ARCH: ${{ matrix.ARCH }}
       KIND_CLUSTER_NAME: kind-${{ matrix.k8s }}
       KUBECONFIG: $HOME/.kube/kind-config-${{ matrix.k8s }}
     steps:
+      - name: Import secrets
+        uses: hashicorp/vault-action@cb841f2c86fb6d07cff94fda240828c1abc5ba43 # v2.7.3
+        id: secrets
+        with:
+          exportEnv: false
+          url: ${{ secrets.VAULT_URL }}
+          role: ${{ secrets.VAULT_ROLE }}
+          method: kubernetes
+          path: kubernetes-ci
```

Comment:
We switched x86 runners to the new cluster as well, right?

Reply:
Not really

I switched this now so we can actually test what happens when we get scheduled on these (and this is how we discovered issue with chart testing and dind mounts)

Technically, we have both old and new runners available now, which one is picked is sort of random, you can see that when ''ingress-runner*' is picked (old one), we fail to get creds from vault

We should decide whether we consider new amd64 runners ready and wish to switch to them fully (not only in this repo), as they're available org-wide as I understand